packages/@expo/cli/src/prebuild/updatePackageJson.ts

```diff
@@ -239,19 +239,19 @@ export function createDependenciesMap(dependencies: any): DependenciesMap {
 }
 
 /**
- * Update package.json scripts - `npm start` should default to `expo
- * start --dev-client` rather than `expo start` after prebuilding, for example.
+ * Updates the package.json scripts for prebuild if the scripts match
+ * the default values used in project templates.
  */
-function updatePkgScripts({ pkg }: { pkg: PackageJSONConfig }) {
+export function updatePkgScripts({ pkg }: { pkg: PackageJSONConfig }) {
   let hasChanged = false;
   if (!pkg.scripts) {
     pkg.scripts = {};
   }
-  if (!pkg.scripts.android?.includes('run')) {
+  if (!pkg.scripts.android || pkg.scripts.android === 'expo start --android') {
```

Comment:
We used a match against "run" so users could keep community CLI scripts in place. The exact match seems a bit fragile.